client/app/screens/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Login from 'react-native-login';
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
You should use import `Icon` from `native-base`— that's what we're using elsewhere.